### PR TITLE
Fix seek

### PR DESCRIPTION
**Pull request by Tommo-L**
- Fix Rocksdb.Seek
- Fix Leveldb.Seek, avoid cross table.

@ProDog plz, try it again.

### CODE REVIEW

File: src/LevelDBStore/IO/Data/LevelDB/Helper.cs [hunk 1]
```diff
@@ -39,5 +41,14 @@ internal static byte[] ToByteArray(this IntPtr data, UIntPtr length)
             Marshal.Copy(data, buffer, 0, (int)length);
             return buffer;
         }
+
+        public static byte[] CreateKey(byte table, byte[] key = null)
```
Comment (shargon) [inline src/LevelDBStore/IO/Data/LevelDB/Helper.cs:45]: We should try to detach RocksDb form LevelDb

Comment (Tommo-L) [reply src/LevelDBStore/IO/Data/LevelDB/Helper.cs:45]: ? This method is only for LevelDb, and there is no dependence between them.

**Comment by ProDog** [comment]
I have tested, it's OK.

File: src/LevelDBStore/IO/Data/LevelDB/Helper.cs [hunk 2]
```diff
@@ -39,5 +41,14 @@ internal static byte[] ToByteArray(this IntPtr data, UIntPtr length)
...
+        {
+            if (key is null) return new[] { table };
```
Comment (shargon) [inline src/LevelDBStore/IO/Data/LevelDB/Helper.cs:47]: or length ==0?

Comment (Tommo-L) [reply src/LevelDBStore/IO/Data/LevelDB/Helper.cs]: Done

**Pull request merged**